frontend/src/layout/header/UserBox.tsx

```diff
@@ -118,8 +116,12 @@ const LoggedIn: React.FC<LoggedInProps> = ({ user }) => {
                 borderRadius: 8,
                 padding: "8px 10px 8px 16px",
                 cursor: "pointer",
-                ":hover": { outline: "2px solid var(--grey80)" },
-                ":focus": { outline: "2px solid var(--accent-color)" },
+                ":hover": {
+                    borderColor: "var(--grey80)",
+                    outline: "1.5px solid var(--grey80)",
+                },
+                ":focus-visible": { borderColor: "var(--accent-color)" },
+                ...focusStyle({ width: 1.5 }),
```

Comment:
Same as previous comment... though this renders fine for me. No wrong pixels. Might still be worth fixing.